lib/noun-project-api.rb

```diff
@@ -1,10 +1,15 @@
 require 'oauth'
 require 'json'
+require 'noun-project-api/connection'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.